roboteam_ai/src/control/ballHandling/DribbleForwards.cpp

```diff
@@ -13,9 +13,10 @@ namespace ai {
 namespace control {
 
 
-RobotCommand DribbleForwards::getRobotCommand(const world::Robot::RobotPtr &r, const Vector2 &targetP,
+RobotCommand DribbleForwards::getRobotCommand(world::Robot::RobotPtr r, const Vector2 &targetP,
         const Angle &targetA) {
-    robot = r;
+
+    robot = std::move(r);
```

Comment:
I personally don't really like std::move here, as you are editing the robotPtr object which becomeso a nullptr now, so if dribbleForwards tries to access the robotptr afterwards is null now. Is there any reason why you made this change?

Reply:
also if this is accessing world directly you are erasing robot pointers in world